### ISSUE TITLE
"js" container gets mime type

### ISSUE THREAD

**Issue by Otto-AA** [solid/node-solid-server#1361]
When the name of a container is a file extension (e.g. js), the parent of it will list it with a mime-type.

For instance if I make this request: 
```javascript
SolidAuthClient.fetch('https://otman.solid.community/public/', {
  method: 'POST',
  headers: {
    link: '<http://www.w3.org/ns/ldp#BasicContainer>; rel="type"',
    slug: 'js',
    'content-type': 'text/turtle' }
})
```
Then `https://otman.solid.community/public/` will contain following:
```text/turtle
js:
    a j:Resource, ldp:BasicContainer, ldp:Container, ldp:Resource;
    terms:modified "2019-11-16T12:48:43Z"^^XML:dateTime;
    st:mtime 1573908523.907;
    st:size 4096.
```
So it says, that the container is a javascript resource. It also works for css and highly likely other file extensions.

**Comment by jaxoncreed** [comment]
Oh, interesting. Good catch. I welcome a pull request for a fix for this, but if not, I will eventually get around to fixing it.

**Comment by jaxoncreed** [comment]
Fixed in #1363
